csrankings-g.csv

```diff
@@ -546,8 +546,6 @@ Golnaz Badkobeh,Goldsmiths University of London,https://www.gold.ac.uk/computing
 Gonçalo N. Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13269,mRRctaMAAAAJ
 Gonçalo Nuno Gomes Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13269,mRRctaMAAAAJ
 Gonçalo Tavares,Universidade de Lisboa,https://fenix.tecnico.ulisboa.pt/homepage/ist13269,mRRctaMAAAAJ
-Gonenc Ercan,Hacettepe University,http://www.gonencercan.com,s7-3H78AAAAJ
-Gönenç Ercan,Hacettepe University,http://www.gonencercan.com,s7-3H78AAAAJ

```

Comment:
Please point to new affiliation if possible.

Reply:
updated